theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -6,13 +6,17 @@ let subjKeywordsApp = new Vue({
     components: {
         VueBootstrapTypeahead
     },
-    delimiters: ['${', '}'],
     data: {
         newKeyword: '',
         resMode: RESOURCE_MODE,
-        resKeywords: RES_KEYWORDS,   // global constant defined in template
+        resKeywords: RES_KEYWORDS, // global constant defined in parent template of subject template
         showIsDuplicate: false,
-        error: ''
+        error: '',
+    },
+    mounted() {
+        // Until Django / VueJS cleaned up across RLP wait for VueJS state to finalize before trying to display dynamic elements
+        document.getElementById('subj-inline-error').style.display = "block";
+        document.getElementById('lst-tags').style.display = "block";
```

Comment:
Wondering why not use jquery $ to get element to set dynamic style here? Is there a disadvantage of using jquery vs document.getEelementById()?

Reply:
That's a great question. I've hinted at this before, but it is now becoming a more concerted effort. We should be reducing library sprawl and more importantly, reducing the burden on how many variations of languages and frameworks devs are required to know to write and review code. VueJS seems like it's very much a part of HS frontend now, bringing it into a reactive paradigm. This overlaps enough with jQuery that we can remove jQuery from most or all places and devs won't need to know jQuery, just JavaScript and some VueJS. Discover has zero jQuery and the library isn't included, (but does exist as a part of the big global scope that is being chipped away and cleaned up).